### PR TITLE
feat: working towards supporting podman in wrapper script

### PR DESCRIPTION
**Pull request by Gowiem**
## what
* Updates `wrapper` to use `DOCKER_EXE` to work towards supporting podman (I don't claim that this works correctly yet)

## why
* Docker is becoming more and more bloatware and can be a huge pain. Podman and others are starting to gain traction and supporting them is of interest on my team.  

## references
* N/A

### CODE REVIEW

File: rootfs/templates/wrapper [hunk 1]
```diff
@@ -197,18 +197,18 @@ function use() {
```
Comment (nitrocode) [inline rootfs/templates/wrapper:197]: Do you also need to update `rootfs/sbin/docker` ?

https://github.com/cloudposse/geodesic/blob/6bd1a5125cd32e9bcf71db0d10bda053dfbc4d69/rootfs/sbin/docker#L10

**Comment by nitrocode** [comment]
@Gowiem friendly ping. I'd love to see podman support in geodesic. The docker app is very slow in comparison.

File: rootfs/templates/wrapper (hunk 1 above)
Comment (nitrocode) [reply rootfs/templates/wrapper:197]: or create a `rootfs/sbin/podman` ?

Comment (Gowiem) [reply rootfs/templates/wrapper:197]: Probably... I'm assuming that is for DinD? That's likely low on my hit list for this PR, but is a good one to track for a future fix.

**Comment by Gowiem** [comment]
@nitrocode yeah, I found that aliasing won't work as well. And you can build + run through podman, but I find a lot of the utility of Geodesic comes with the wrapper as it simplifies everything for non-Geodesic experts who need to run it. 

We have some Makefile work that utilizes this code that I believe got podman working, but it was painful and I (obviously) dropped it. If you want to chat through that, I'm happy to show it to you, but it'd likely be more confusing if I just shared a bunch of Makefile code here. 

I'll also add this draft PR to our open source board and we'll circle back around to cleaning this up at some point. I'm buying the new Macbooks once they launch for my FTE and myself -- we'll definitely come back around at that point because my ideal is that we don't even need to install Docker onto those machines at all 😁 🤞

**Pull request closed without merging**